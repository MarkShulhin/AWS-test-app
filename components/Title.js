import styled from 'styled-components';

const Title = styled.h1`
  color: papayawhip;
  font-size: 50px;
`;

export default Title;
