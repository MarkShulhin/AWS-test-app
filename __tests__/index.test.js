/* eslint-env jest */
import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

import App from '../pages/index.js';
import Title from '../components/Title';

describe('With Enzyme', () => {
  it('should render App with styled title', () => {
    const wrapper = shallow(<App />);

    expect(wrapper.find(Title).exists()).toBe(true);
  });
});

describe('With Snapshot Testing', () => {
  it('should render App that shows ssr hello world', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
