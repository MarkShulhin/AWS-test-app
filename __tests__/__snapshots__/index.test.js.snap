// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing should render App that shows ssr hello world 1`] = `
<h1
  className="sc-bdVaJa bqcVBL"
>
  Server side rendered "Hello World!"
</h1>
`;
